src/normalizations/normalize-message.ts

```diff
@@ -146,14 +244,17 @@ export const normalizeMailMessageFromSoap = (
 			participants: m.e ? map(m.e || [], normalizeParticipantsFromSoap) : undefined,
 			tags: getTagIds(m.t, m.tn),
 			parts: m.mp ? map(m.mp || [], normalizeMailPartMapFn) : undefined,
+			// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+			// @ts-ignore
+			attachments: m.mp ? getAttachmentsFromParts(m.mp) : undefined,
```

Comment:
you can set attributes size, contentType and name of AttachmentPart as optional. so that the ts-ignore can be removed